index.html

```diff
@@ -38,5 +38,26 @@ <h2 class="text-container" id="sdsection">
       <img src="arrow-button.svg" class="arrow-button"><a href="#"></a></img>
       </div>
     </section>
+    <section class="works">
+      <h1 class="recent-works">My Recent Works</h1>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div>
```

Comment:
- [ ] Please use `display: flex` on each card which is your `div class = "card">  element to ensure that the contents of each project or card is aligned with the flexbox :100: